.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseMergedDateTimeParser.cs

```diff
@@ -292,6 +321,7 @@ public List<DateTimeParseResult> DateTimeResolutionForSplit(DateTimeParseResult
 
             var islunar = val.IsLunar;
             var mod = val.Mod;
+            var typeOutput = DetermineDateTimeType(slot.Type, hasMod: !string.IsNullOrEmpty(mod));
```

Comment:
What's typeOutput ?

Reply:
This is not newly added, moved this code from above.
With modifier, the output type might not be the type in the original resolution result.
For example, if the resolution type is "date", with modifier, the output type should be "daterange".
Code comment added to describe this.

Reply:
outputType then, instead of typeOutput.